sdk/sdk_test.go

```diff
@@ -52,14 +55,21 @@ func TestNew_ShouldCreateSDK(t *testing.T) {
 
 func Test_ShouldCreateNewSDK_NoCredentials(t *testing.T) {
 	// When
-	sdk, err := sdk.New(goodPlatformEndpoint)
+	sdk, err := sdk.New(goodPlatformEndpoint,
+		sdk.WithPlatformConfiguration(sdk.PlatformConfiguration{
+			"platform_issuer": "https://example.org",
+		}),
+	)
 	// Then
 	require.NoError(t, err)
 	assert.NotNil(t, sdk)
 }
 
 func TestNew_ShouldCloseConnections(t *testing.T) {
 	sdk, err := sdk.New(goodPlatformEndpoint,
+		sdk.WithPlatformConfiguration(sdk.PlatformConfiguration{
```

Comment:
is it worth it for one of these tests to sanity check the new `.PlatformIssuer()` method?

Reply:
Updated one of the tests to make sure we test that now.